azkaban-web-server/src/main/resources/conf/azkaban-users.xml

```diff
@@ -1,7 +0,0 @@
-<azkaban-users>
-	<user username="azkaban" password="azkaban" roles="admin" groups="azkaban" />
-	<user username="metrics" password="metrics" roles="metrics"/>
```

Comment:
@suvodeep-pyne Never mind. I thought some azkaban-user.xml file might be loaded when Intellij runs the main method of az-solo-server.